types/index.d.ts

```diff
@@ -0,0 +1,141 @@
+// TypeScript Version: 3.0
+
+declare namespace parseEntities {
+  interface ParseEntitiesOptions<WC = any, TC = any, RC = any> {
```

Comment:
https://devblogs.microsoft.com/typescript/announcing-typescript-3-4/#type-checking-for-globalthis

```suggestion
  interface ParseEntitiesOptions<WC = globalThis, TC = globalThis, RC = globalThis> {
```